### ISSUE TITLE
Consider adding a lesson on Rmarkdown

### ISSUE THREAD

**Issue by naupaka** [datacarpentry/R-genomics#47]
We could port over (e.g.) the existing [lesson from SWC's r-novice-gapminder](https://github.com/swcarpentry/r-novice-gapminder/blob/gh-pages/_episodes_rmd/15-knitr-markdown.Rmd) as a place to start. I think knitr/Rmarkdown is a great way to tie back the analyses done with scripts and on the command line into a document that can be handed off to a collaborator.

**Comment by naupaka** [comment]
For the Stanford DC-genomics workshop, we had [an Rmd template](https://github.com/naupaka/stanford-r-genomics/blob/master/lessons_rmd/template.Rmd) for what one of these might look like, incorporating the parsing of the VCF files and some munging.

**Comment by nlhuong** [comment]
I agree it would be very helpful to include a lesson on rmarkdown, as most people analyzing genomics data would like to generate reports with text comments and visualization together with the code.

I think it would also be beneficial to include a section in http://www.datacarpentry.org/R-genomics/00-before-we-start.html explaining the differences in document types RStudio offers, e.g. what are the differences and similarities between R Script/R Notebook/ R Markdown (even R presentation). 
Following this how to generate html / pdf reports and what are the arguments one can set to get a nice looking report, e.g. toc, biocstyle etc.

**Comment by naupaka** [comment]
@nlhuong thanks for the feedback. We definitely plan to include a knitr/rmarkdown module in the R Genomics lessons that are in the early stages of development. Stay tuned!
